sources/background_scripts/image_context_menu.js

```diff
@@ -94,19 +106,27 @@
         });
     }
 
-    define(["common/configuration"],
-    function(configuration_module)
-    {
-        configuration = configuration_module;
-
-        browser.storage.onChanged.addListener((changes, area) =>
+    define(["common/configuration", "background_scripts/notifications"],
+        function(configuration_module)
```

Comment:
`notifications_module` should be a parameter here (next to `configuration_module`).